Makefile

```diff
@@ -172,7 +172,7 @@ kind-e2e-chart-tests: ## Run helm chart e2e tests
 e2e-test-binary:  ## Build binary for e2e tests.
 	@build/run-in-docker.sh \
 		MAC_OS=$(MAC_OS) \
-		ginkgo build ./test/e2e
+		GOBIN=/tmp/bin go generate tools/ginkgo.go && ls /tmp/bin/ginkgo && /tmp/bin/ginkgo build ./test/e2e
```

Comment:
the ls is required?

Reply:
Not required.
For test purposes, I shall eliminate the 'ls' following the rectification of the CI.